genestack_client/files_util.py

```diff
@@ -748,3 +749,72 @@ def create_dataset(self, name, dataset_type, children, parent=None):
             if parent is None:
                 parent = self.get_special_folder(SpecialFolders.MY_DATASETS)
             return self.invoke('createDataset', parent, name, dataset_type, children)
+
+    def find_metainfo_related_terms(
+            self,
+            file_filter,
+            dictionary_accession,
+            relationship_label,
+            transitive,
+            key,
+            prefix_match=False,
```

Comment:
same as in the core: using a enum might be better, because `prefix_match=False` doesn't tell anything by itself and requires consulting the documentation